project/plugins.sbt

```diff
@@ -6,5 +6,6 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.6")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "latest.release")
 addSbtPlugin("uk.co.josephearl" % "sbt-verify" % "0.3.0-SNAPSHOT")
 addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.12")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
```

Comment:
Interesting, do you intend it for local use only, or should we include it in the build pipeline?

Reply:
I propose we should (but not necessarily with changes in this PR). It gives you a very good idea of how old your libraries are.